### DISCUSSION TITLE
-LOCAL_NETWORK environment variable

### DISCUSSION THREAD

**Discussion by derekcentrico** [haugene/docker-transmission-openvpn#2520] [General]
I'm curious how the -LOCAL_NETWORK environment variable is built/created into this docker.  Can anyone point me to the that?

I'm trying to play with a separate container and wondering how to bake that in potentially.

**Comment by edgd1er** [comment]
I'm not sure I understood the question. 
the LOCAL_NETWORK is set at run time with `-e LOCAL_NETWORK=...`or environment in docker-compose.yml, as stated in README.md

The variable is used in start.sh to open ports in firewall (if enabled) and add route from/to local_network.

**Reply by derekcentrico** [reply]
Yeah so I'm trying to see how that is baked in to try to use it in something else.  Goal is to separate out non-torrent related dockers to use a different VPN instead.  I got a docker container going fine and like all the dockers are communicating remotely, but there is no bridge if you will to the local network so they are inaccessible locally.  Curious how that works so I can newbie my way around and see what I may be able to hack up.
